### DISCUSSION TITLE
Crash on LogIn with apple id

### DISCUSSION THREAD

**Discussion by CrisLDev** [hyochan/react-native-iap#1650] [Q&A] [answered]
Help pls, I'm getting this error when I make click in a product.
[https://i.stack.imgur.com/kQmkH.png](url)
I'm using 6.0.0 library version and only fail in apple.

**Comment by idlework** [comment, marked as answer]
@CrisLDev If you read the error, it says your textinput is not declared. This is not a `react-native-iap` issue. Hopefully you already found this out.
